brave/brave-core
# Rewrite brave ads sanitize code in Rust.

Resolves https://github.com/brave/brave-browser/issues/30020



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Check that the visited web pages classification works
- Check that text embedding processing works: https://github.com/brave/brave-core/pull/14393
